hawc/apps/assessment/views.py

```diff
@@ -695,12 +695,18 @@ def get_context_data(self, **kwargs):
         iveps = self.model.ivendpoint.related.related_model.objects.get_qs(
             self.assessment.id
         ).count()
-        alleps = eps + os + mrs + iveps
+        epiv2_designs = apps.get_model("epiv2", "Design").objects.get_qs(self.assessment.id).count()
+        epiv2_data_extractions = (
```

Comment:
this is counting epiv2 data extractions, but it's a list of epiv2 outcomes

Reply:
@shapiromatron

Reply:
Thanks, that seems off, I will fix